testsuite/features/build_validation/containerization/proxy_as_pod_basic_tests.feature

```diff
@@ -20,14 +20,12 @@ Feature: Register and test a Containerized Proxy
 
   Scenario: Pre-requisite: Unregister Salt minion in the traditional proxy
     Given I am on the Systems overview page of this "sle15sp4_minion"
-    When I stop salt-minion on "sle15sp4_minion"
-    And I follow "Delete System"
+    When I follow "Delete System"
     Then I should see a "Confirm System Profile Deletion" text
     When I click on "Delete Profile"
-    Then I wait until I see "Cleanup timed out. Please check if the machine is reachable." text
-    When I click on "Delete Profile Without Cleanup" in "An error occurred during cleanup" modal
     And I wait until I see "has been deleted" text
-    Then "sle15sp4_minion" should not be registered
+    And I wait until Salt client is inactive on "sle15sp4_minion"
+    Then "sle15sp4_minion" should not be registered￼
```

Comment:
Is there a trailing unknown character or is it just a rendering error?

Reply:
Fixed, thanks